package.json

```diff
@@ -6,8 +6,9 @@
   "scripts": {
     "docs": "jsdoc2md src/sheetFunctions.js > doc/sheet_functions.md",
     "push": "clasp push -f",
-    "pretest": "eslint .",
-    "test": "mocha"
+    "pretest": "eslint . ; npm run build",
+    "test": "mocha",
+    "build": "node generators/getMetric.js generateGetMetric"
```

Comment:
`generate` or something like this, not `build`

Reply:
You're right, I've just changed it